### PR TITLE
Update egg-garrys-mod.json

### PR DESCRIPTION
**Pull request by Nobody196**
Add server.cfg, workshop.lua and more variables

### CODE REVIEW

File: database/seeds/eggs/source-engine/egg-garrys-mod.json [hunk 1]
```diff
@@ -49,6 +49,51 @@
             "user_viewable": 0,
             "user_editable": 0,
             "rules": "required|string|max:20"
+        },
+        {
```
Comment (DaneEveritt) [inline database/seeds/eggs/source-engine/egg-garrys-mod.json]: Won't this break if you don't provide a workshop ID?

File: database/seeds/eggs/source-engine/egg-garrys-mod.json [hunk 2]
```diff
@@ -49,6 +49,51 @@
...
+            "env_variable": "GM_NAME",
+            "default_value": "sandbox",
+            "user_viewable": 1,
+            "user_editable": 1,
+            "rules": "required|string"
+        },
+        {
+            "name": "Max Players",
+            "description": "The maximum amount of players allowed on your game server.",
+            "env_variable": "PLYS",
```
Comment (DaneEveritt) [inline database/seeds/eggs/source-engine/egg-garrys-mod.json]: This is a terrible environment key name, theres no reason to shorten it like that.

File: database/seeds/eggs/source-engine/egg-garrys-mod.json [hunk 3]
```diff
@@ -49,6 +49,51 @@
...
+            "default_value": "32",
+            "user_viewable": 1,
+            "user_editable": 1,
+            "rules": "required|integer|max:128"
+        },
+        {
+            "name": "Tickrate",
+            "description": "The tickrate defines how fast the server will update each entities location.",
+            "env_variable": "TICK",
```
Comment (DaneEveritt) [inline database/seeds/eggs/source-engine/egg-garrys-mod.json]: This is also a bad name.

File: database/seeds/eggs/source-engine/egg-garrys-mod.json [hunk 4]
```diff
@@ -49,6 +49,51 @@
...
+            "env_variable": "WORKSHOP_API",
+            "default_value": "",
+            "user_viewable": 1,
+            "user_editable": 1,
+            "rules": "nullable"
+        },
+        {
+            "name": "Gamemode",
+            "description": "The gamemode of your server.",
+            "env_variable": "GM_NAME",
```
Comment (DaneEveritt) [inline database/seeds/eggs/source-engine/egg-garrys-mod.json]: This should just be `GAMEMODE`

**Comment by Nobody196** [comment]
Let me makes some changes real quick o.o

File: database/seeds/eggs/source-engine/egg-garrys-mod.json [hunk 5]
```diff
@@ -49,6 +49,51 @@
...
+            "name": "Workshop ID",
+            "description": "The ID of your workshop collection (the numbers at the end of the URL)",
+            "env_variable": "WORKSHOP_ID",
+            "default_value": "",
+            "user_viewable": 1,
+            "user_editable": 1,
+            "rules": "nullable|integer"
+        },
+        {
+            "name": "Workshop API Key",
```
Comment (TrixterTheTux) [inline database/seeds/eggs/source-engine/egg-garrys-mod.json]: Not used anymore

File: database/seeds/eggs/source-engine/egg-garrys-mod.json (hunk 1 above)
Comment (Nobody196) [reply database/seeds/eggs/source-engine/egg-garrys-mod.json]: It'll spit an error but won't stop startup of the server

**Comment by Nobody196** [comment]
Do I need to mark it as changed anywhere?

**Comment by ZeProf2Code** [comment]
@Nobody196 Try to commit the same thing again? Idk

**Pull request merged**